### ISSUE TITLE
Не появляются зависимости в файле package.json.

### ISSUE THREAD

**Issue by YBkhrv** [YBkhrv/my-learn-path#4]
Здравствуйте,я сейчас нахожусь на строчке "Once the installation is complete, you should be able to see your project dependencies listed inside your package.json file:"  в https://nextjs.org/learn/foundations/from-react-to-nextjs/getting-started-with-nextjs.

- Появилось сообщение в командной строке  "npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead"- исправила при помощи https://stackoverflow.com/questions/72401421/message-npm-warn-config-global-global-local-are-deprecated-use-loc.
Но список зависимостей не появился в  файле package.json.

- Попробовала "npm install react react-dom next" в консоли:
up to date, audited 18 packages in 4s
2 packages are looking for funding
 run `npm fund` for details
found 0 vulnerabilities
Список зависимостей не появился в  файле package.json.

- Потом попробовала в консоли:
npm fund
Yulia
+-- https://opencollective.com/browserslist
|   `-- caniuse-lite@1.0.30001364
`-- https://opencollective.com/postcss/
    `-- postcss@8.4.5

https://opencollective.com/browserslist и https://opencollective.com/postcss/ -это сайты для финансовой поддержки ПО.
По поиску в Интернете "-- caniuse-lite@1.0.30001364" нашла на https://yarnpkg.com/package/caniuse-lite справа команду "yarn add caniuse-lite",но она ""$" не является внутренней или внешней
командой, исполняемой программой или пакетным файлом."
Как установить -- caniuse-lite@1.0.30001364 и -- postcss@8.4.5?Как сделать,чтобы появились зависимости в package.json?Или можно идти дальше по обучению?

Заранее спасибо за ответ.

**Comment by alexbaumgertner** [comment]
Подскажите версию Node и версию OS (Windows/Linux/Mac)

**Comment by YBkhrv** [comment]
Node v16.15.1
Windows 10.

**Comment by YBkhrv** [comment]
Пошла дальше по обучению с пустым package.json:

- Остановилась на строчке" To confirm everything works, you can view your app by running npm run dev inside your terminal and navigating to [localhost:3000](http://localhost:3000/) in the browser. Then, make a small change to the code and save it.".
Пишет "C:\Users\Migrane>npm run dev
npm ERR! Missing script: "dev"
npm ERR!
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Migrane\AppData\Local\npm-cache\_logs\2022-08-01T08_30_07_068Z-debug-0.log"
Попробовала https://stackoverflow.com/questions/41129815/how-to-resolve-npm-run-dev-missing-script-issues - не помогло.

- скопировала содержание
 // package.json
   {
    "scripts": {
        "dev": "next dev"
    },
     // "dependencies": {
     //   "next": "^11.1.0",
     //   "react": "^17.0.2",
     //   "react-dom": "^17.0.2"
     // }
   }

понимаю,что версии next,react,react-dom разные,но хотя бы попробовала,тоже не помогло.

**Comment by alexbaumgertner** [comment]
Покажите ваш package.json


Missing script: "dev" -- говорит о том, что у вас нет dev в секции scripts. Должно быть так:

```json
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint"
  },
```

**Comment by alexbaumgertner** [comment]
+ как решение -- сможете поставить WSL https://docs.microsoft.com/en-us/windows/wsl/install

**Comment by YBkhrv** [comment]
- Установила по вложенной ссылке https://docs.microsoft.com/en-us/windows/wsl/install-manual (steps 1  и 6(в Downloading distributions первая ссылка Ubuntu)). Какие теперь шаги?Заново  пройти https://nextjs.org/learn/foundations/about-nextjs ?

- package.json заполнила исходя из 

 https://stackoverflow.com/questions/41129815/how-to-resolve-npm-run-dev-missing-script-issues - не помогло.

скопировала содержание
// package.json
{
"scripts": {
"dev": "next dev"
},
// "dependencies": {
// "next": "^11.1.0",
// "react": "^17.0.2",
// "react-dom": "^17.0.2"
// }
}

понимаю,что версии next,react,react-dom разные,но хотя бы попробовала,тоже не помогло.

Без заполнения было так:

// package.json { }

**Comment by alexbaumgertner** [comment]
Да, нужно в WSL пройти заново все шаги — установить nodejs,  установить nextjs и дальше.
